environments/icds-cas/public.yml

```diff
@@ -116,6 +116,10 @@ couch_dbs:
 
 TWO_FACTOR_GATEWAY_ENABLED: True
 
+s3_blob_db_enabled: yes
+s3_blob_db_url: "https://mowcdmum.ipstorage.tatacommunications.com"
+s3_blob_db_s3_bucket: 'mowcdmum-c1'
```

Comment:
should this wait on https://github.com/dimagi/commcare-hq/pull/21726? or did we get s3 api access?